app/code/Magento/Theme/Model/View/Design.php

```diff
@@ -100,19 +100,6 @@ public function __construct(
         $this->objectManager = $objectManager;
     }
 
-    /**
-     * Set package area
-     *
-     * @param string $area
-     * @return $this
-     */
-    public function setArea($area)
-    {
-        $this->_area = $area;
```

Comment:
It seems like this property shouldn't be used anymore and should be removed. What do you think?

Reply:
I'm not sure remove that property can make something else break

Reply:
ok. let's leave it as is

Reply:
oh no i have removed that @@